### PR TITLE
Created PriorityQueueEg.java

### PR DESCRIPTION
**Pull request by iamvs-2002**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Implemented an example of Priority Queue data structure in Java.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1689 
(Write your answer here.)

### CODE REVIEW

**Comment by iamvs-2002** [comment]
This is for DWOC - DevScript Winter of Code.
Thankyou!

**Comment by iamvs-2002** [comment]
@shraddhavp Please help me resolve this DeepSource error. It certainly is not any error in the code.

**Comment by shraddhavp** [comment]
@iamvs-2002  dont worry it is a deepsource check fail,it is a new feature where implementation is still in progress ,it will soon be fixed ,it is not the error of PR dont worry

**Comment by iamvs-2002** [comment]
Okay, thanks for the clarification.

File: Java/ds/PriorityQueueEg.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import java.util.Iterator;
+import java.util.PriorityQueue;
+import java.lang.*;
+import java.util.Scanner;
+
+//Priority Queue data structure is a special queue in which the elements are present not as per FIFO order but as per the natural elements or any xyz comparator used during queue creation.
```
Comment (HarshCasper) [inline Java/ds/PriorityQueueEg.java]: Break this down to make it readable

File: Java/ds/PriorityQueueEg.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+public class PriorityQueueEg {
+    
+    private static Scanner sc = new Scanner(System.in);
+
+    public static void main(String args[])
```
Comment (HarshCasper) [inline Java/ds/PriorityQueueEg.java:21]: Shift your core Code Logic into another function. Use main() just for your driver code.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Make README changes as well

File: Java/ds/PriorityQueueEg.java (hunk 1 above)
Comment (iamvs-2002) [reply Java/ds/PriorityQueueEg.java]: Ohh Okay!

File: Java/ds/PriorityQueueEg.java (hunk 2 above)
Comment (iamvs-2002) [reply Java/ds/PriorityQueueEg.java:21]: Oh yea...that way the code would look more readable.

**Comment by HarshCasper** [comment]
Add corresponding README changes as well!

**Comment by iamvs-2002** [comment]
@HarshCasper Done!

**Review by HarshCasper — APPROVED** [review]

**Comment by iamvs-2002** [comment]
@HarshCasper Kindly add the dwoc label as the core team says that the label must be both on the pr and the issue.

**Pull request merged**